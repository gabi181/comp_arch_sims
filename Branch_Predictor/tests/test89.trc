8 2 15 2 local_history global_tables using_share_mid
0x5A7D7718 T 0x7797CC5C
0x21D71534 T 0x18D666C8
0x5A7D7718 N 0x7797CC5C
0x5A7D7718 T 0x7797CC5C
0x33F3A110 T 0x683CA794
0x33F3A110 T 0x683CA794
0x7E449ECC N 0x31C93E40
0x33F3A110 T 0x683CA794
0x21D71534 T 0x18D666C8
0x33F3A110 T 0x683CA794
0x21D71534 T 0x18D666C8
0x7E449ECC T 0x31C93E40
