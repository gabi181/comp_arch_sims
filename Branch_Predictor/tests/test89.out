0x5a7d7718 N 0x5a7d771c
0x21d71534 N 0x21d71538
0x5a7d7718 T 0x7797cc5c
0x5a7d7718 T 0x7797cc5c
0x33f3a110 N 0x33f3a114
0x33f3a110 T 0x683ca794
0x7e449ecc N 0x7e449ed0
0x33f3a110 N 0x33f3a114
0x21d71534 T 0x18d666c8
0x33f3a110 N 0x33f3a114
0x21d71534 T 0x18d666c8
0x7e449ecc T 0x31c93e40
flush_num: 6, br_num: 12, size: 392b
